qiskit/aqua/algorithms/minimum_eigen_solvers/qaoa/var_form.py

```diff
@@ -52,7 +50,8 @@ def __init__(self, cost_operator: WeightedPauliOperator,
             TypeError: invalid input
         """
         super().__init__()
-        cost_operator = op_converter.to_weighted_pauli_operator(cost_operator)
+        # TODO MatrixToPauli converter
```

Comment:
Yes, it's blocked on https://github.com/Qiskit/qiskit-terra/pull/3932, which will make this free. If that's not in by release, we'll implement our own quickly.

Reply:
We need the converter anyway just to have, but when https://github.com/Qiskit/qiskit-terra/pull/4071 is merged, this will rely on that rather than converting to Pauli, because it will perform much better.

Reply:
This is done - both because the QAOA ansatz now accepts MatrixOps and because we now can convert to sums of PauliOps.